package.json

```diff
@@ -20,20 +25,24 @@
     "popper.js": "^1.14.3",
     "postcss-loader": "^3.0.0",
     "prettier": "^1.17.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.0",
```

Comment:
Are we using `react-scripts` anywhere?

Reply:
I don't know whet that appears there. It is removed now.